### DISCUSSION TITLE
Dani Grant - jam.dev

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#267] [Announcements]
- https://twitter.com/thedanigrant
- https://jam.dev/
- https://jam.dev/blog/jam-raises-8-9m-series-a-led-by-ggv-capital/

### Past

- [x] I was looking into your background and your early career is fascinating. You had a business internship (in china), a design internship, a software internship, an ad internship and then you ended up as a project manager at cloudflare. It seems like you put yourself through a "work in every part of the company" guantlet. How did that end up happening and do you think those experiences across the "stack" have helped you be a CEO?

#### Product

- [x] What is jam and why would I want to use it?
- [x] What were some failed products you made only the way?
- [x] I read that what jam currently is wasn't where it started. What product iterations do you go through to get here?
- [x] What is instant replay?
- [x] How does JamGPT work? What type of bugs can it help me fix?
- [x] What experiences at cloudflare led you to creating Jam?

#### Running a Company

- [x] Congrats on the Series A! You have some heaving hitters in the group like Figma, RauchG among many others. [Ask something about funding]
- [x] Do you have any advice for first time founders pitching to investors?
- [x] What are some lessons you were told as a startup founder that didn't quite pan out?
- [x] As a CEO, what do you think you have to do to grow and cultivate a good team culture?

### Future

- [x] Now that you have all that money what the plan? What's the next big milestone for Jam?
- [x] What do you think the future of collaboration looks like?

**Comment by hipstersmoothie** [comment]
https://github.com/DaniGuardiola/classy-ink
https://twitter.com/0xca0a/status/1762069894932160943

**Comment by just-be-dev** [comment]
https://github.com/lsd-rs/lsd
https://github.com/sindresorhus/ky

**Comment by danigrant** [comment]
tooltips:
https://slack.com/help/articles/206329808-Google-Calendar-for-Slack
https://typefully.com/
https://www.youtube.com/@mreflow
